src/services/reviews.service.js

```diff
@@ -0,0 +1,29 @@
+import Reviews from '../database/models/reviews.model';
+
+async function getReviews(productId) {
+  const reviews = await Reviews.findAll({ where: { productId } });
+  return reviews;
+}
+
+async function addReview(data) {
+  const addedReview = await Reviews.create(data);
```

Comment:
How about performing some validation checks here to ensure that the required data is provided before creating the review.

Reply:
There is a **joi** validator check in reviews route.

Reply:
Yeah, I just noticed